src/app/cli/Cargo.toml

```diff
@@ -101,6 +101,7 @@ tracing-log = "0.2"
 tracing-bunyan-formatter = "0.3"
 
 # Utils
+assert_cmd = "2.0.13"
```

Comment:
This library is only needed for testing purposes, so we don't want to link it to `kamu` as a normal dependency when building the binary. Optimizer will throw it away, but it will still slow down the compile time.

There is a special section `[dev-dependencies]` for libraries that only tests need.

Reply:
thank you, done